sn_networking/src/driver.rs

```diff
@@ -394,6 +394,8 @@ impl NetworkBuilder {
 
         // Gossipsub behaviour
         let gossipsub_config = libp2p::gossipsub::ConfigBuilder::default()
+            // disable sending to ALL_PEERS subscribed to a topic, which is the default behaviour
+            .flood_publish(false)
```

Comment:
this may cause a `subscriber` miss a msg?
it it is not known by ALL publisher ?

with the `flood` disabled`, will `forwarding` still get undertaken?